cmd/messaging-tool/receive.go

```diff
@@ -31,13 +31,21 @@ var receiveCmd = &cobra.Command{
 	Run:   runReceive,
 }
 
-func callback(m client.Message, destination string) (err error) {
+func callback(message client.Message, destination string) (err error) {
+	if message.Err != nil {
+		glog.Errorf(
+			"Can't subscribe to destination '%s': %s",
+			destinationName,
+			message.Err.Error(),
+		)
+		return
```

Comment:
The callback has return type `error` - should this maybe return an error obj?

Reply:
Currect !!! Thanks ! fixing now !